ledger/simulation/tracer.go

```diff
@@ -127,6 +128,9 @@ func (tracer *evalTracer) populateInnerTransactions(txgroup []transactions.Signe
 	applyDataOfCallingTxn.EvalDelta.InnerTxns = append(applyDataOfCallingTxn.EvalDelta.InnerTxns, txgroup...)
 }
 
+// execTraceStack keeps track of the call stack from top level transaction to the current inner txn that contains latest TransactionTrace
+var execTraceStack []*TransactionTrace
+
```

Comment:
Is the endpoint limited to one simulation at a time?

Reply:
yes, one at a time, and I suppose even we are simulating multiple groups, we are still doing it one at a time?